packages/uniwind/src/core/native/store.ts

```diff
@@ -80,11 +90,50 @@ class UniwindStoreBuilder {
         }
     }
 
+    private resolveMediaQueryVars(dependencies: Set<StyleDependency>) {
```

Comment:
@Brentlok Kindly review this:

The coderabbitai has reviewed most part of the code, but I'll need an in-dept review of this method resolving media queries at runtime